helm/g2p-sandbox/values.yaml

```diff
@@ -578,6 +578,9 @@ ph-ee-engine:
       annotations:
         deployTime: "{{ .Values.deployTime }}"
 
+  post_installation_job:
```

Comment:
please change the indentation one level left

Reply:
All the services come under ph-ee-chart, so one indentation is fine for all except Account_mapper ( it is independent of ph-ee-engine)